cowprotocol/cowswap
# feat(twap): update ComposableCoW contract

# Summary

Fixes #2639

New addresses:
 - ExtensibleFallbackHandler	0x4e305935b14627eA57CBDbCfF57e81fd9F240403
 - ComposableCoW	0xF487887DA5a4b4e3eC114FDAd97dc0F785d72738
 - TWAP	0x910d00a310f7Dc5B29FE73458F47f519be547D3d
 - CurrentBlockTimestampFactory	0x0899c7DC280363d263Cc954506134F249CceC4a5
 
 What changed:
 1. Changed the call of `ComposableCoW.create` to `ComposableCoW.createWithContext` 
 2. Added `appData` to TWAP order
 3. Added usage of `CurrentBlockTimestampFactory` to set a block timestamp as TWAP order startTime 

  # To Test

1. Should create a TWAP order as before
2. Should check and change Fallback handler as before
3. Should display a valid appData in Explorer for part-orders
